public/app/features/alerting/unified/Analytics.ts

```diff
@@ -8,3 +11,30 @@ export const LogMessages = {
   cancelSavingAlertRule: 'user canceled alert rule creation',
   successSavingAlertRule: 'alert rule saved successfully',
 };
+
+// logInfo from '@grafana/runtime' should be used, but it doesn't handle Grafana JS Agent and Sentry correctly
+export function logInfo(message: string, context: Record<string, string | number> = {}) {
+  if (config.grafanaJavascriptAgent.enabled) {
+    agent.api.pushLog([message], {
+      level: GrafanaLogLevel.INFO,
+      context: { ...context, module: 'Alerting' },
+    });
+  }
+}
+
+export function withPerfLogging<TFunc extends (...args: any[]) => Promise<any>>(
```

Comment:
Could we use another more explanatory name for this method?, not sure if something like `withPerformaceLogging`?